packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -17,3 +18,30 @@ export const correctStackForCrossDomainError = (serializedError: any, userInvoca
 
   return reifiedError
 }
+
+export const omitUnserializableValues = (value) => {
+  const { isDom, isDocument, isWindow, isJquery } = $dom
+
+  // There are probably some things we aren't catching here.
+  // We probably want to bubble this up to the user over a generic error to get a better idea of what couldn't be serialized
+  if (isDom(value)
+  || isDocument(value)
+  || isWindow(value)
```

Comment:
`document` and `window` are included in the `isDom` check so they can be removed.